test/lib/organization/constructor.js

```diff
@@ -18,19 +18,172 @@
 //
 // ----------------------------------------------------------------------------
 
+const web3 = require('../../test_lib/web3.js');
+const EventDecoder = require('../../test_lib/event_decoder.js');
+const Utils = require('../../test_lib/utils.js');
+
 const Organization = artifacts.require('Organization');
 
 contract('Organization.constructor()', async (accounts) => {
 
-  let owner = accounts[0];
+  const zeroAddress = '0x0000000000000000000000000000000000000000';
+
+  const owner = accounts[3];
+  const admin = accounts[4];
+  const workers = [
+    accounts[5],
+    accounts[6],
+    accounts[7],
+    accounts[8],
+  ];
+  let expirationHeight;
+
+  beforeEach(async function () {
+    let currentBlockHeight = await web3.eth.getBlockNumber();
+    expirationHeight = currentBlockHeight + 10;
+  });
+
+  it('should construct with all parameters given', async () => {
```

Comment:
This should also assert `workers` and it's `expirationHeight`.

Reply:
See `it('should register all given worker addresses with the given expiration height')` lines 94 ff.